packages/system/foundationdb-operator/values.yaml

```
@@ -0,0 +1,4 @@
+fdb-operator:
+  globalMode:
+    enabled: true
+  nodeReadClusterRole: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)